release-notes/opensearch-release-notes-2.6.0.md

```diff
@@ -0,0 +1,539 @@
+# OpenSearch and OpenSearch Dashboards 2.6.0 Release Notes
+
+## Release Highlights
+
+### New Features
+
```

Comment:
* With the new Simple Schema for Observability, OpenSearch introduces a standardized approach to accessing data from different sources. The schema supports a structured definition for major analytics and observability signals, including logs, traces, and metrics, conforming to OpenTelemetry standards.
* You can now create, view, and manage data  streams directly from the index management user interface. Admins also gain the ability to perform manual rollover operations and force merges for indexes or data streams from the UI.
* You can now use multiple indexes or index patterns to create threat detectors, rather than just a single source. Many detectors now include out-of-the-box dashboards to help you identify broader patterns in your security logs.
* The search backpressure feature can now cancel queries at the coordinator  level, offering more efficient protection against traffic surges that result from a small number of resource-intensive queries.
* You can now add maps to dashboard panels within OpenSearch Dashboards without leaving the Dashboards environment.